server/views/pages/adjudicationForReport/hearingTab.njk

```diff
@@ -32,7 +32,7 @@
         {% if item.hearing %}
           {% set hearingNumber = hearingNumber + 1 %}
           {{ hearingTable(hearingNumber, item.hearing, item.outcome, readOnly, reportNo, latestHearingId) }}
-          {% if item.outcome %} 
+          {% if item.outcome and item.hearing.outcome.code != HearingOutcomeCode.COMPLETE %}
```

Comment:
extra check for when the hearing is complete and finding is NOT_PROCEED - without this, the 'not proceed' table will show under the hearing, which isn't what we want here (but is when the outcome is NOT_PROCEED in other circumstances)